### DISCUSSION TITLE
What is the purpose of the backup function?

### DISCUSSION THREAD

**Discussion by jebbgrenham** [leonlatsch/Photok#315] [General]
Without a restore-from-backup option, I'm curious as to what the idea was to add a backup function if those backups are functionally unable to be used? Is the restore-from-backup something in the development plan which just hasn't been gotten around to yet?

**Comment by leonlatsch** [comment]
Actually, you can restore backups. Just open the import menu and choose "Backup".

**Comment by jebbgrenham** [comment]
In settings I see the "Backup" option; but selecting it opens a file browser to create a backup. No import menu anywhere. Above Backup is "Reset safe" and below it I see the "Delete Imported Files" selector. I'm on 1.6.0 which is the latest build on google play.

**Comment by leonlatsch** [comment]
Its not in the settings. Its in the normal manu that you also use to add photos or videos

![image](https://github.com/leonlatsch/Photok/assets/50983026/9a58e7bd-7cd4-4c32-95ea-1d9b71c5b349)

**Comment by jebbgrenham** [comment]
Ahhhhh! Well I'm an idiot. I would have never thought to look there and somehow I've never noticed that before. I expected it to be either an option for a fresh install (ie restore from backup as an initial gallery) or next to the backup option in settings). Thank you.
